includes/Modules/Analytics_4.php

```diff
@@ -68,6 +69,7 @@ final class Analytics_4 extends Module
 	public function register() {
 		$this->register_scopes_hook();
 
+		add_action( 'googlesitekit_analytics_handle_provisioning_callback', $this->get_method_proxy( 'handle_provisioning_callback' ) );
```

Comment:
Can we add a test for this? We should be able to use our `FakeHttpClient` to mock responses to the Google API. It would be good to assert that the relevant entities are created and settings updated when this fires.

Reply:
Added.